test/PowerNotifications/PowerTests.cpp

```diff
@@ -0,0 +1,205 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "winrt/Microsoft.ProjectReunion.h"
+
+using PowerEventHandle =
+winrt::Windows::Foundation::EventHandler<winrt::Windows::Foundation::IInspectable>;
+
+using namespace winrt::Microsoft::ProjectReunion;
+
+namespace ProjectReunionPowerTests
+{
+    constexpr auto TIMEOUT = 2000;
+    class PowerTests
+    {
+    public:
+        BEGIN_TEST_CLASS(PowerTests)
+            TEST_CLASS_PROPERTY(L"ActivationContext", L"PowerTests.dll.manifest")
+            TEST_CLASS_PROPERTY(L"ThreadingModel", L"MTA")
+            END_TEST_CLASS()
+
+            TEST_METHOD(GetBatteryStatus)
+        {
+            auto stat = PowerManager::BatteryStatus();
+            VERIFY_ARE_EQUAL(stat, BatteryStatus::NotPresent);
+        }
+
+        TEST_METHOD(GetPowerSupplyStatus)
+        {
+            auto stat = PowerManager::PowerSupplyStatus();
+            VERIFY_ARE_EQUAL(stat, PowerSupplyStatus::Adequate);
+        }
+
+        TEST_METHOD(GetRemainingChargePercent)
+        {
+            auto stat = PowerManager::RemainingChargePercent();
+            VERIFY_ARE_EQUAL(stat, 100);
+        }
+
+        TEST_METHOD(CompositeBatteryStatusCallback)
+        {
+            // This test can only be run on systems/VM with a battery
+            // The values will need to be changed to reflect real-time values
```

Comment:
`//TODO:...` ?

Reply:
Not really. The comment is just to guide for systems that can run it.